app/consumer-democracy/app.go

```diff
@@ -244,28 +248,35 @@ func New(
 	appOpts servertypes.AppOptions,
 	baseAppOptions ...func(*baseapp.BaseApp),
 ) *App {
-	encodingConfig := makeEncodingConfig()
+	encodingConfig := MakeTestEncodingConfig()
 
 	appCodec := encodingConfig.Codec
 	legacyAmino := encodingConfig.Amino
 	interfaceRegistry := encodingConfig.InterfaceRegistry
 	txConfig := encodingConfig.TxConfig
 
+	// ABCI++, v50
+	voteExtOp := func(bApp *baseapp.BaseApp) {
```

Comment:
do we need to extend a pre-commit vote since we dont have an arbitrary data and the dummy imp says that it should be only used for testing purposes? if we want to leave it in the app just for testing, its ok (same goes for consumer and provider apps)

Reply:
Unfortunately, I don't have much context about this.
Since vote-extension is already set in `v5.0.0`,  I think we can afford to have it here as well.